.github/workflows/tests-on-pr-push-and-merge.yml

```diff
@@ -40,10 +40,12 @@ jobs:
         with:
           token: ${{ secrets.BROADBOT_GITHUB_TOKEN }}
       - name: Set up AdoptOpenJDK 11
+        id: setup_jdk
```

Comment:
nit: Are you just putting IDs on everything, or was this needed for something specific? Other workflows are only doing it when it's referenced somewhere.

Reply:
Yes, I just put IDs on everything as a best practice. I'm not using it for anything here.